Antidote/ActiveCallViewController.m

```diff
@@ -446,7 +465,7 @@ - (void)callMenuButtonPressed
 - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath
 {
     [self.dataSource activeCallController:self resumePausedCallSelectedAtIndex:indexPath];
-    self.tableViewOfPausedCalls.hidden = YES;
+    self.pauseCallsContainer.hidden = YES;
     [self.tableViewRefreshTimer invalidate];
```

Comment:
Hm, here you should call `hideTableViewOfPausedCalls` instead of hiding and invalidating timer.